app/controllers/idv/in_person/address_controller.rb

```diff
@@ -43,6 +46,24 @@ def extra_view_variables
         }
       end
 
+      # update Idv::DocumentCaptureController.step_info.next_steps to include
+      # :ipp_address instead of :ipp_ssn in delete PR
+      def self.step_info
+        Idv::StepInfo.new(
+          key: :ipp_address,
+          controller: self,
+          next_steps: [:ipp_ssn],
+          preconditions: ->(idv_session:, user:) { idv_session.ipp_state_id_complete? },
+          undo_step: ->(idv_session:, user:) do
+            flow_session[:pii_from_user][:address1] = nil
+            flow_session[:pii_from_user][:address2] = nil
+            flow_session[:pii_from_user][:city] = nil
+            flow_session[:pii_from_user][:zipcode] = nil
+            flow_session[:pii_from_user][:state] = nil
```

Comment:
@svalexander, since the state_id step also sets these fields, I wonder if we shouldn't clear them here. What do you think?

Reply:
I think it's ok for now, especially since we're not yet including this step as a next step anywhere, so undo_step won't be called. And then if we always clear starting after SsnController for the in person steps that edit pii_from_user and ssn, it won't be a problem. But we'll have to give this more thought and see if we can come up with a better solution.